seatunnel-formats/seatunnel-format-json/src/main/java/org/apache/seatunnel/format/json/ogg/OggJsonDeserializationSchema.java

```diff
@@ -102,7 +102,7 @@ public OggJsonDeserializationSchema(
     @Override
     public SeaTunnelRow deserialize(byte[] message) throws IOException {
         throw new SeaTunnelJsonFormatException(
-                CommonErrorCodeDeprecated.JSON_OPERATION_FAILED,
+                CommonErrorCode.JSON_OPERATION_FAILED,
                 String.format("Failed to deserialize JSON '%s'.", new String(message)));
     }
```

Comment:
I think you should create a method in `CommonError` to create `SeaTunnelRunTimeException` with `CommonErrorCode.JSON_OPERATION_FAILED` just like other exception. Then throw exception by `throw CommonError.jsonOperationFailed(content)`; Do not throw new `CommonErrorCode` with such way.

Reply:
ack